CelesteNet.Client/Components/CelesteNetPlayerListComponent.cs

```diff
@@ -350,7 +351,7 @@ public CelesteNetPlayerListComponent(CelesteNetClientContext context, Game game)
 
                     // move all the right column's elements to the right via Dyn.X
                     for (int i = switchedSidesAt; i < list.Count; i++)
-                        list[i].Dyn.X = sizeColumn.X + 15f;
+                        list[i].Dyn.X = sizeColumn.X + scale * 30f;
```

Comment:
And for anyone wondering:
Snip just put the 30f in there because it looked about right, and [according to my analysis](https://gist.github.com/RedFlames/a78cca45e130bf6da65dee5d06c0d19b) of all this padding and spacing junk, it might well be the value that should go there.